Exercise_C.js

```diff
@@ -14,5 +14,13 @@ See README.md if you need help running your code.
 */
 
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+  if (i % 3 == 0 && i % 5 == 0) {
+    console.log("Hong Kong");
+  } else if (i % 3 == 0) {
+    console.log("Hong");
+  } else if (i % 5 == 0) {
+    console.log("Kong");
+  } else {
+    console.log(i);
+  }
 }

```

Comment:
Hello Anson,
Great work on this exercise and alternative method that could be used is 
`for (let i = 1; i <= 30; i++) {
  if (i % 15 == 0) {
    console.log("Hong Kong");
  } else if (i % 3 == 0) {
    console.log("Hong");
  } else if (i % 5 == 0) {
    console.log("Kong");
  } else {
    console.log(i);
  }
}`